code/example1.py

```diff
@@ -0,0 +1,10 @@
+# Hundreds of lines of code above this
+
+urls = []
+for path in paths:
+  url = 'https://'
+  url += domain
+  url += path
+  urls.append(url)
```

Comment:
Here's a cleaner way to refactor this bit of code:

```
urls = ['https://' + domain + path for path in paths]
```

Here is an example: https://www.tutorialspoint.com/python-list-comprehension

Reply:
Thanks ! that's a neat construct! I'll implement this now.